sdk/servicebus/service-bus/test/utils/aadUtils.ts

```diff
@@ -1,21 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import { EnvVarNames, getEnvVars } from "./envVarUtils";
-
-import { loginWithServicePrincipalSecret } from "@azure/ms-rest-nodeauth";
-
-const aadServiceBusAudience = "https://servicebus.azure.net/";
+import { EnvironmentCredential } from "@azure/identity";
 
 export async function getTokenCredentialsFromAAD() {
-  const env = getEnvVars();
-  const tokenCreds = await loginWithServicePrincipalSecret(
-    env[EnvVarNames.AAD_CLIENT_ID],
-    env[EnvVarNames.AAD_CLIENT_SECRET],
-    env[EnvVarNames.AAD_TENANT_ID],
-    {
-      tokenAudience: aadServiceBusAudience
-    }
-  );
-  return tokenCreds;
+  return new EnvironmentCredential();
```

Comment:
I don't see the `getTokenCredentialsFromAAD()` helper method being used anywhere anymore. Can we remove it?

Reply:
Right, updated it